tests/Feature/TransliteratorBuilderTest.php

```diff
@@ -43,13 +43,12 @@ public function testTransliteratorBuilderWithScriptNamesInNativeLanguagesWithRep
         $transliterator = (new TransliteratorBuilder())
             ->toASCII()
             ->replace('\u002D', '\u0020')
-            ->replace('ʌ', 'e')
-            ->replace('Ɔ', 'O')
+            ->IPAToEnglishApproximation()
             ->keep((new Filter())->addRange('a', 'z')->addRange('A', 'Z')->addChar(' '))
         ;
 
         static::assertSame(
-            'Naxi Geba naci geba Na Khi Ggo baw Nakhi Geba',
+            'Naxi Geba naci guba Na Khi Ggo baw Nakhi Geba',
```

Comment:
@PrinsFrank What do you think about the other two "Geba"-s?

Reply:
These are all variations of the same two words but with slightly different spellings. I think in this case it doesn't matter if this stays as an 'e' in the standards package either, so I'll leave it be.